packages/admin-panel/src/authentication/ProfileButton.js

```diff
@@ -3,22 +3,13 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 import React from 'react';
-import { useLocation } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import { ProfileButton as BaseProfileButton, ProfileButtonItem } from '@tupaia/ui-components';
 
-export const ProfileButton = ({ user, isBESAdmin }) => {
-  const location = useLocation();
-  const inVizBuilder = location.pathname.startsWith('/viz-builder');
+export const ProfileButton = ({ user }) => {
   const ProfileLinks = () => (
     <>
       <ProfileButtonItem to="/profile">Edit Profile</ProfileButtonItem>
-      {isBESAdmin && !inVizBuilder && (
-        <ProfileButtonItem to="/viz-builder/new">Visualisation Builder</ProfileButtonItem>
-      )}
-      {inVizBuilder && (
-        <ProfileButtonItem to="/dashboard-items">Exit Visualisation Builder</ProfileButtonItem>
-      )}
```

Comment:
Might still be worth keeping this `Exit Visualisation Builder` button? Personally I never used it, and generally chose to navigate back to the Admin Panel by clicking the Admin Panel logo. But I know some people do use it...

Reply:
(Also could keep but maybe move it somewhere else as it doesn't seem appropriate to be in the `Profile` dropdown)

Reply:
>(Also could keep but maybe move it somewhere else as it doesn't seem appropriate to be in the Profile dropdown)

Yep I thought this too, but we already have the "back to admin panel" link in the top left. Is this enough or do we want another one? It's minor so if it's hard to find a new place lets just leave it in the profile menu.

Reply:
Let's just leave it in the menu for now? I know changing it might cause some confusion for folks and I don't know if I know the best UX here. Personally I'd be happy to remove it entirely as you have, but might wanna get Dave's feedback first...

Reply:
Checked with Slack, people seem to be ok with it being removed. David G didn't respond